### ISSUE TITLE
Add justifyContent and alignItems props to Flex in v1, to facilitate migration to v2

### ISSUE THREAD

**Issue by jes708** [jxnblk/styled-system#358]
Use case: pkg-search consumes a number of different common components from various teams.  We are about to start using one (@pcln/summary-of-charges) that was built with design system v2. We have too many dependencies which still rely on v1 to upgrade our app to v2 at the moment.  It seems that the only issue in the summary-of-charges component is that `justifyContent='space-between'` does not work.  If we could add this as an acceptable prop to Flex v1, this would ease the migration to v2, and allow us to use the new summary-of-charges component without having to upgrade ds in our pkg-search for now.

TL;DR `Flex` v1 should accept `justifyContent` and `alignItems` props
